tests/e2e/utils/flows.js

```diff
@@ -589,43 +587,6 @@ export const merchantWCP = {
 		} );
 	},
 
-	openDisputeDetails: async ( disputeDetailsLink ) => {
-		await Promise.all( [
-			page.goto( WC_ADMIN_BASE_URL + disputeDetailsLink, {
-				waitUntil: 'networkidle0',
-			} ),
-			uiLoaded(),
-		] );
-		await uiLoaded();
-	},
-
-	openChallengeDispute: async () => {
-		await Promise.all( [
-			evalAndClick(
-				// eslint-disable-next-line max-len
-				'div.transaction-details-dispute-details-body div.transaction-details-dispute-details-body__actions button.components-button.is-primary'
-			),
-			page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-			uiLoaded(),
-		] );
-	},
-
-	openAcceptDispute: async () => {
```

Comment:
I've removed this helper function and moved the logic inline to simplify the code – this was only used in one test.